foxglove/cmd/utils.go

```diff
@@ -228,3 +230,12 @@ func maybeConvertToRFC3339(timestamp string) (string, error) {
 	}
 	return parsed.Format(time.RFC3339), nil
 }
+
+func AuthIsApiKey() (bool, error) {
+	token := viper.Get("bearer_token")
+	if token == nil {
+		dief("Auth is not setup. Run `foxglove auth login` or `foxglove auth configure-api-key` to continue.")
```

Comment:
nit: I think "setup" should be two words. Or maybe something like "Not signed in."